homeassistant/components/stream/__init__.py

```diff
@@ -464,3 +470,25 @@ def get_diagnostics(self) -> dict[str, Any]:
 def _should_retry() -> bool:
     """Return true if worker failures should be retried, for disabling during tests."""
     return True
+
+
+STREAM_OPTIONS_SCHEMA: Final = vol.Schema(
```

Comment:
Is this meant to be the schema for the input to `create_schema` or the input into pyav? I was thinking the former, however from the naming I was not clear since above we call stream_options what is passed into pyav.

Reply:
Sorry - it's confusing because currently the only option is really passed through verbatim. From the naming and the usage, yes it is meant to be the former. I will adjust the naming in create_stream.